specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -367,211 +429,224 @@
+          }
         },
-        "message": {
-          "description": "The error message.",
-          "type": "string"
+        "parameters": [
+          {
+            "$ref": "./billingSubscription.json#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "./types.json#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingSubscriptionAliasListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./types.json#/definitions/ErrorResponse"
+            }
+          }
         },
-        "details": {
-          "description": "Detailed error message explaining the error.",
-          "type": "string"
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
         }
       }
     },
-    "SubscriptionTransferValidationErrorCode": {
-      "type": "string",
-      "description": "Error code of the transfer validation response.",
-      "enum": [
-        "BillingAccountInactive",
-        "CrossBillingAccountNotAllowed",
-        "DestinationBillingProfileInactive",
-        "DestinationBillingProfileNotFound",
-        "DestinationBillingProfilePastDue",
-        "DestinationInvoiceSectionInactive",
-        "DestinationInvoiceSectionNotFound",
-        "InsufficientPermissionOnDestination",
-        "InsufficientPermissionOnSource",
-        "InvalidDestination",
-        "InvalidSource",
-        "MarketplaceNotEnabledOnDestination",
-        "NotAvailableForDestinationMarket",
-        "ProductInactive",
-        "ProductNotFound",
-        "ProductTypeNotSupported",
-        "SourceBillingProfilePastDue",
-        "SourceInvoiceSectionInactive",
-        "SubscriptionNotActive",
-        "SubscriptionTypeNotSupported"
-      ],
-      "x-ms-enum": {
-        "name": "subscriptionTransferValidationErrorCode",
-        "modelAsString": true
-      }
-    },
-    "ErrorDetails": {
-      "description": "The details of the error.",
-      "properties": {
-        "code": {
-          "description": "Error code.",
-          "type": "string",
-          "readOnly": true
-        },
-        "message": {
-          "description": "Error message indicating why the operation failed.",
-          "type": "string",
-          "readOnly": true
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingSubscriptions/aliases/{aliasName}": {
+      "get": {
+        "tags": [
+          "BillingSubscriptionsAliases"
+        ],
+        "operationId": "BillingSubscriptionsAliases_Get",
+        "description": "Gets a subscription by its alias ID.  The operation is supported for seat based billing subscriptions.",
```

Comment:
"Gets a subscription by its alias". Alias is always an ID right? It is not an object